src/collection/components/CollectionOrBundleEdit.tsx

```diff
@@ -168,6 +175,7 @@ const CollectionOrBundleEdit: FunctionComponent<
 	useWarningBeforeUnload({
 		when: unsavedChanges,
 	});
+	const [contributors, setContributors] = useState<Contributor[]>();
```

Comment:
either use:
```
	const [contributors, setContributors] = useState<Contributor[]>([]);
```

or 
```
	const [contributors, setContributors] = useState<Contributor[] | null>(null);
```

so typescript can catch any "object may be undefined" errors